frontend/src/components/page-lifeGroups/topLGSummary.js

```diff
@@ -3,59 +3,68 @@ import { HighlightedParagraph } from "../shared/highlightedParagraph";
 import { SecondaryButtonLink } from "../Button";
 import { StaticImage } from "gatsby-plugin-image";
 
-const TopLGSummary = () => (
-  <div>
-    <h1 className="text-center text-[40px] pt-[2.1875rem] lg:pt-20 font-bold mb-[2.25rem] lg:mb-0">
-      LIFE GROUPS
-    </h1>
+const TopLGSummary = ({ lgSignupsRef }) => {
+  const handleScroll = () => {
+    if (lgSignupsRef.current) {
+      lgSignupsRef.current.scrollIntoView({ behavior: "smooth" });
+    }
```

Comment:
you don't have to do this. you can use id and use the with hash.  this is a default browser behavior. for example the container id="lg-anchor" and the URL just needs to have #lg-anchor.

Reply:
resolved